projects/core/src/i18n/i18next/i18next-translation.service.ts

```diff
@@ -76,18 +94,23 @@ export class I18nextTranslationService implements TranslationService {
     return this.i18next.loadNamespaces(chunkNames);
   }
 
-  /**
-   * Returns a fallback value in case when the given key is missing
-   * @param key
-   */
-  protected getFallbackValue(key: string): string {
-    return isDevMode() ? `[${key}]` : this.NON_BREAKING_SPACE;
+  protected getFallbackValue(keyOrKeys: string | string[]): string {
```

Comment:
```suggestion
  /**
   * Returns a fallback value in case when the all given keys are missing
   */
  protected getFallbackValue(keyOrKeys: string | string[]): string {
```

Reply:
Changed